mistletoe/block_token.py

```diff
@@ -152,7 +153,8 @@ def __init__(self, lines):
 
 class Heading(BlockToken):
     """
-    Heading token. (["### some heading ###\\n"])
+    ATX heading token. (["### some heading ###\\n"])
+    This is a leaf block token. Its children are inline (span) tokens.
     Boundary between span-level and block-level tokens.
```

Comment:
My interpretation of the sentence is that it explains that the children are span-level tokens, but not in a very clear way. So yes, I agree that it can be removed.

Reply:
Thanks.